wayat/frontend/ios/firebase_app_id_file.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file has critical data, please delete it and add it to the frontend gitignore

Reply:
Fix